tests/e2e/cucumber/steps/app-files/resource.ts

```diff
@@ -63,6 +63,36 @@ Given(
   }
 )
 
+When(
+  '{string} downloads the following file(s) using batch action',
+  async function (this: World, stepUser: string, stepTable: DataTable) {
```

Comment:
please use the already existing download step for that and extend it with an `via` instruction or use the table steps for that.